docs/governance/community-policies/numbering-format.md

```diff
@@ -0,0 +1,13 @@
+# Numbering Format
+
+## Taxonomy
+
+This is the proposed numbering format used in the taxonomy service features.
+
+`CCC.TX.CAT.XX.YYY`
```

Comment:
It occurs to me that we can have a unique service identifier such as `OBST` for Object Storage, and it mitigates the need for inclusion of a category number.

Also, if we use C/F to specify if the final value is a control or feature ID, then we mitigate the need for `TX`. 

**CCC.`Service** Name Shorthand`.F`Feature Incrementer`**

example: `CCC.OBST.F01`

What do you think of this idea?

Reply:
I like it. Though it does raise a question for me. CCC is my first FINOS project so I am not too familiar with what else it does. But are there are any other FINOS projects that do something similar? 

Should a standard identifier format be something that is consistent across the FINOS level? Or do you think that would be too wide a scope to capture.

e.g.
<FINOS_PROJECT_NAME>.<PROJECT_SPECIFC_IDS>.<FEATURE_INCREMENTER>

FINOS_PROJECT_NAME and FEATURE_INCREMENTER are a standard approach taken by all FINOS projects, and individiual projects are free to specify their PROJECT_SPECIFIC_IDS?

Reply:
I doubt it, but let's ask around.

@robmoffat @karlmoll Do you have visibility to answer this? Has FDC3, TraderX, or anything else created a unique identifier pattern that we could draw inspiration from?

Reply:
The closest thing I can think is where in FDC3 we named all of our different tests, since they exercised different features of FDC3.  e.g. https://github.com/finos/FDC3/blob/main/toolbox/fdc3-conformance/Open-Tests.md

You can see on here we have `AOpensBWithContext1` or `AFailsToOpenB3` etc.  Since you'll also be building tests in CCC/CFI you'll definitely need some identifiers to separate them out.  The one piece of advice I can give is that numbers aren't super-helpful and we did get into some renumbering issues at times, which you don't get with identifiers.